dispatcher/dispatcher.go

```diff
@@ -37,24 +40,16 @@ type (
 var (
 	// DefaultConfig is the default config
 	DefaultConfig = Config{
-		ActionChanSize:             5000,
-		BlockChanSize:              1000,
-		BlockSyncChanSize:          400,
+		ActionChanSize:    5000,
+		BlockChanSize:     1000,
+		BlockSyncChanSize: 400,
+		ConsensusChanSize: 1000,
+		MiscChanSize:      1000,
```

Comment:
4844 plan to use the miscChan? if that's the case, consider just use the feature name like `blobChan`?

Reply:
I think blobTx can use actionChan